src/App.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import Cell from "./components/Cell";
+
+const App = () => {
+  const [cells, setCells] = useState(["", "", "", "", "", "", "", "", ""]);
+  const [go, setGo] = useState("circle");
```

Comment:
I suppose this is also from the guide. In programming, if you repeat a value more than once, it means that you probably want to store it in a variable instead and use that variable.
E.g. my suggestion here would be to add something like `const circle = 'circle'` and then use the variable everywhere you need it. Same for other similar values.

This approach have multiple benefits but most important are that the chance of typing the work incorrectly is minimal in the case of variable and you only need to update one place if you want to change the value everywhere.